sys-devel/clang-common/clang-common-16.0.6-r2.ebuild

```diff
@@ -153,6 +153,25 @@ src_install() {
 			@gentoo-common.cfg
 		EOF
 	done
+
+	if use kernel_Darwin; then
+		cat >> "${ED}/etc/clang/gentoo-common.cfg" <<-EOF || die
+			# Gentoo Prefix on Darwin
+			-Wl,-search_paths_first
+			-Wl,-rpath,${EPREFIX}/usr/lib
+			-L ${EPREFIX}/usr/lib
+			-isystem ${EPREFIX}/usr/include
+			-isysroot ${EPREFIX}/MacOSX.sdk
+		EOF
+		if use bootstrap-prefix ; then
+			cat >> "${ED}/etc/clang/gentoo-common.cfg" <<-EOF || die
+				-Wl,-rpath,${EPREFIX}/../usr/lib
```

Comment:
What would or should this point to?  This looks kind of worrysome.  I don't have a `usr` next to my `$EPREFIX`

Reply:
This is during stage2, where EPREFIX is EPREFIX/tmp, so it will point at libraries of stage3 during one of rebuilds

Reply:
ewl... this can't be done from env or something?

In the very least ensure EPREFIX ends with tmp and ${EPREFIX%/tmp}/usr/lib exists?

Reply:
Should I use another env var here instead of EPREFIX, which would be set from the bootstrap-prefix.sh for the stage 2? like, `EPREFIX_FOR_STAGE3`

Reply:
I see what your problem is.   Can you at least add a hint/reminder in the bootstrap-prefix block that that USE-flag is only meant to be set when EPREFIX = EPREFIX/tmp or something?

Reply:
Good idea. Done.